### PR TITLE
feat(db/locales): ruRU - Add missing quest_request_items_locale for quest 

### PR DESCRIPTION
**Pull request by Bogir**
id 11001-12000.

## SOURCE:

Taken from https://github.com/TrinityCore/TrinityCore/issues/27860

## Tests Performed:

- Compared with https://wotlk.evowow.com

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by LukasVolgger — CHANGES_REQUESTED** [review]
Please use single quotes instead of double ones at all strings to keep a line.

**Comment by LukasVolgger** [comment]
@Bogir thx for this. I left a review and also please change your PR's title to this standards: https://www.conventionalcommits.org/en/v1.0.0/ example: `feat(db/locales): ...`

File: data/sql/updates/pending_db_world/rev_1646834195276318302.sql [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+(11016, 'ruRU', 'Нам нужны эти шкуры на подкладку для брони и прочие технические нужды.', 0),
+(11023, 'ruRU', 'Ну, как бомбардировка?', 0),
+(11026, 'ruRU', 'Мы так думаем, ты $Gиспользовал:использовала; кристалл изгнания, чтобы послать целую кучу этих демонов Пылающего Легиона паковать вещички?!', 0),
+(11027, 'ruRU', 'Демоны из этого лагеря постоянно ремонтируют транспортера.$B$BГахку нужна темная руна, чтобы сделать выкованную кристалл с темными рунами.$B$BЕсли у маленького $R нет темной руны, поговори с Кронком. Он делает одну темную руну каждый день!', 0),
+(11029, 'ruRU', 'Книга, $N. Ты $Gдостал:достала; её?', 0),
+(11030, 'ruRU', 'Ты уже $Gдобыл:добыла; этот магический настой? Для Торкуса-младшего он имеет большое значение.', 0),
+(11036, 'ruRU', 'Это поставка от старины Орока?', 0),
+(11051, 'ruRU', '$GТакой:Такая; $Gопытный:опытная; $Gпрофессионал:профессионалка; с легкостью сможет изгнать демонов. Правда же, $R?', 0),
+(11056, 'ruRU', 'Ты вернулся, $N. Я уже начал думать, что ты меня бросил.', 0),
+(11058, 'ruRU', 'Вы уже научились пользоваться апекситовой реликвией? И нам, и Ша'тарским Небесным Стражам к северу интересно, что вы о них узнали.', 0),
```
Comment (LukasVolgger) [inline data/sql/updates/pending_db_world/rev_1646834195276318302.sql:18]: The problem  now is, that you have to escape some of the single qutes like this `\'` or `''`

File: data/sql/updates/pending_db_world/rev_1646834195276318302.sql [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+(11145, 'ruRU', 'Удалось ли вам освободить пленников в деревне Черного Копыта?', 0),
+(11146, 'ruRU', 'Ты поймал этих хищников?', 0),
+(11147, 'ruRU', 'Вы выпустили хищников в деревне Черного Копыта?', 0),
+(11153, 'ruRU', 'Мы снова говорим? Это должно означать, что ты это $Gсделал:сделала;! Ты кое-что $Gпоказал:показала; этим гнилым пиратам Северного моря!', 0),
+(11154, 'ruRU', 'Пожалуйста, скажи мне, что ты $Gпринес:принесла; гуано!', 0),
+(11155, 'ruRU', 'Нет ничего вкуснее нежного мяса черпорога, срезанного с боков!', 0),
+(11156, 'ruRU', 'Удалось ли вам остановить атаки Зловещего Тотема?', 0),
+(11157, 'ruRU', 'Как продвигаются ваши усилия?', 0),
+(11164, 'ruRU', 'Поторопись, $Gдружище:подружка;!$B$BМожет, пошлют меня в Зул'Аман одного, если я в ближайшее время не отдам ему эти бивни!', 0),
+(11166, 'ruRU', 'Я верю, что мои созданные вручную маскировки обеспечили вам адекватную защиту в Зул'Амане.$B$BВы обследовали эти места для меня? Или мне начать искать замену?', 0),
```
Comment (LukasVolgger) [inline data/sql/updates/pending_db_world/rev_1646834195276318302.sql:54]: Here too

File: data/sql/updates/pending_db_world/rev_1646834195276318302.sql [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+(11490, 'ruRU', 'Иди и смотри в шар, $Gсмертный:смертная;', 0),
+(11491, 'ruRU', 'Не беспокойся о нем, $N. Он переживет это ...', 0),
+(11492, 'ruRU', 'Да, $C, должен признаться, я не был уверен, что увижу тебя снова.', 0),
+(11496, 'ruRU', 'Вы активировали кристалл, $N?', 0),
+(11499, 'ruRU', 'Терраса Магистров, наверное, очень опасное место – слишком уж близко оно к Солнечному Колодцу.$B$BНадеюсь, вы не передумали?', 0),
+(11500, 'ruRU', 'Вы еще не разобрались с сестрами.', 0),
+(11513, 'ruRU', 'Ты $Gдобыл:добыла; контейнеры с маной?', 0),
+(11515, 'ruRU', 'Эти уроды - позор для нашей расы. Уничтожь их, $N.', 0),
+(11516, 'ruRU', 'Готово, $N? Мы должны помешать дальнейшим силам Легиона усилить Кель'тас.', 0),
+(11520, 'ruRU', 'Честное слово, $N, мне не до разговоров.$B$BПросто дай мне знать, когда добудешь корни.', 0),
```
Comment (LukasVolgger) [inline data/sql/updates/pending_db_world/rev_1646834195276318302.sql:190]: And here - 225

**Review by LukasVolgger — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1646834195276318302.sql (hunk 3 above)
Comment (Bogir) [reply data/sql/updates/pending_db_world/rev_1646834195276318302.sql:190]: Fixed

**Review by LukasVolgger — APPROVED** [review]

**Pull request merged**